desc/compute/_basis_vectors.py

```diff
@@ -11,8 +11,8 @@
 
 from desc.backend import jnp
 
+from ..utils import cross, dot, safediv
```

Comment:
why do we not do `desc.utils` here? or is this equivalent?

Reply:
it's equivalent. this is what Pycharm's automated refactoring chose.